### DISCUSSION TITLE
Physical driving update?

### DISCUSSION THREAD

**Discussion by Firezombie232** [Okabintaro/SubmersedVR#127] [General]
I see that some people have added physical driving as an option (In the pull requests) but the latest version doesn't seem to support it. Will this mod be updated to fix that?

**Comment by Okabintaro** [comment]
Yes I plan to release 0.2 with the current state of the mod soon.

**Reply by Firezombie232** [reply]
Awesome! How soon would you say that is coming? Thank you very much for your hardwork on the mod btw.
